cumulus/client/consensus/aura/src/collators/lookahead.rs

```diff
@@ -522,11 +525,49 @@ async fn cores_scheduled_for_para(
 		},
 	};
 
+	// Get `AsyncBackingParams` from runtime
+	let (tx, rx) = oneshot::channel();
+	let request = RuntimeApiRequest::AsyncBackingParams(tx);
+	overseer_handle
```

Comment:
Might be better to use `load_abridged_host_configuration` for the async backing params so the collator works without full node. It should also avoid spawning the runtime in favor of a faster direct storage read.

Reply:
Done